core/vibes/soul/sections/sign-up-section/index.tsx

```diff
@@ -1,27 +0,0 @@
-import { DynamicForm, DynamicFormAction } from '@/vibes/soul/primitives/dynamic-form';
```

Comment:
I'm not sure if we should remove this yet.

Reply:
@jorgemoya Since the sign up page is so simple, we're able to replace the page to use the DynamicFormSection, so we don't need these imports.

Reply:
Gotcha. Sorry was out of the loop!